tests/functional/api/test_groups.py

```diff
@@ -116,6 +115,12 @@ def test_group_labels(group):
     assert label.description == "baz"
     assert len(group.labels.list()) == 1
 
+    label.new_name = "comp:Flat Export"
+    label.save()
+    assert label.name == "comp:Flat Export"
+    label = group.labels.get("comp:Flat Export")
+    assert label.name == "comp:Flat Export"
```

Comment:
I'm ok to merge this as is, but maybe we could use our own example here instead to indicate what we're testing.

Maybe `"scope::Complex Label"` or something like that? I'll leave it up to you :)

Reply:
Sure I can do that.

Reply:
Done